app/assets/stylesheets/products.scss

```diff
@@ -0,0 +1,167 @@
+// Place all the styles related to the products controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.slider {
+  height: 540px !important;
+}
+.slides {
+  height: 500px !important;
+  min-height: 500px !important;
+  // width: 500px !important;
+  li, li.active {
+    z-index: 0 !important;
+  }
+}
+
+.waves-effect {
+  z-index: 0 !important;
+}
+
+.product-description-body {
+  margin-top: 3em;
+}
+.product-view-image {
+  margin: 0 auto;
+}
+.product-name {
+  font-size: +1;
+}
+
+.price {
+  color: #2e7d32;
+  font-size: 1.2em;
+  width: 8em;
+  .old-price{
+    color: #730E15;
+    text-decoration: line-through;
+    font-size: smaller;
+    font-weight: bold;
+  }
+} 
+
+.rating-counts {
+  width: 100%;
+  margin: 10px 0 30px;
+  float: left;
+}
+
+.counts {
+  margin: 1.1em 0 0 1em;
+  font-size: 1em;
+  /*font-family: "Arial Black";*/
+  color: #000;
+}
+
+.rating {
+  margin: 0;
```

Comment:
Properties should be ordered direction, font-size, margin, max-width, unicode-bidi